lib/styles/v2/components/_tag.scss

```diff
@@ -0,0 +1,43 @@
+$solid-indigo: #5E5CE6;
+$solid-indigo-bg: #EAF3FC;
```

Comment:
```suggestion
```

Both colors are added in `variables.scss`. Also, while adding new colors, please follow the same naming as in figma.